src/server.js

```diff
@@ -1,10 +1,86 @@
 const express = require('express');
-
+var http = require('http');
+var env = require('require-env');
+//var express = require('express');
+var bodyParser = require('body-parser');
 const app = express();
+const mysql = require('mysql');
+var getStudent;
 const port = process.env.PORT || 5000;
 
+
+var con = mysql.createConnection({
```

Comment:
this is the kind of thing that would go in a '.env' file and shouldn't be pushed to a online repository. Theoretically, anyone would now be able to connect to your sql instance.